plugins/neo4j-adapter/src/main/java/org/polypheny/db/adapter/neo4j/util/NeoStatements.java

```diff
@@ -538,7 +538,15 @@ static LiteralStatement literal_( PolyValue value ) {
     }
 
     static LiteralStatement string_( PolyValue value ) {
-        return new LiteralStatement( value == null || value.isNull() ? null : "'" + value + "'" );
+        // TODO: do we need to implement more sophisticated way to escape strings?
+        if ( value == null || value.isNull()) {
+            return new LiteralStatement( null );
+        }
+        String quote = "'";
+        if ( value.toString().contains( "'" ) ) {
+            return new LiteralStatement( "\"" + value + "\"" );
```

Comment:
What happens when values contain `"` and `'`?

Reply:
Previously the generated bind code only uses single quotes to represent Cypher strings. Therefore, when a Cypher string contains single quotes, an exception will be triggered. For example, CREATE (:N {msg: "I'd like some tea"}).

Reply:
I didn't really consider the mix of quotes in a string. Will do a fix now and add some test cases